newton/solvers/mujoco/solver_mujoco.py

```diff
@@ -638,6 +638,75 @@ def update_dof_properties_kernel(
     dof_frictionloss[worldid, dof_in_env] = joint_friction[tid]
 
 
+@wp.kernel
+def update_geom_properties_kernel(
+    shape_collision_radius: wp.array(dtype=float),
+    shape_mu: wp.array(dtype=float),
+    shape_ke: wp.array(dtype=float),
+    shape_kd: wp.array(dtype=float),
+    shape_size: wp.array(dtype=wp.vec3f),
+    shape_transform: wp.array(dtype=wp.transform),
+    shape_to_geom: wp.array(dtype=wp.int32),
+    shapes_per_env: int,
+    up_axis: int,
+    # outputs
+    geom_rbound: wp.array2d(dtype=float),
+    geom_friction: wp.array2d(dtype=wp.vec3f),
+    geom_solref: wp.array2d(dtype=wp.vec2f),
+    geom_size: wp.array2d(dtype=wp.vec3f),
+    geom_pos: wp.array2d(dtype=wp.vec3f),
+    geom_quat: wp.array2d(dtype=wp.quatf),
+):
+    """Update geom properties from Newton shape properties."""
+    tid = wp.tid()
+    worldid = tid // shapes_per_env
+    shape_in_env = tid % shapes_per_env
+
+    geom_idx = shape_to_geom[shape_in_env]
+    if geom_idx < 0:  # Invalid mapping
+        return
+
+    # Update bounding radius
+    geom_rbound[worldid, geom_idx] = shape_collision_radius[tid]
+
+    # Update friction (slide, torsion, roll)
+    mu = shape_mu[tid]
+    geom_friction[worldid, geom_idx] = wp.vec3f(mu, 0.05 * mu, 0.05 * mu)  # Use scaled values for torsion/roll
```

Comment:
Instead of these hard-coded numbers, can you use the following `Model` parameters instead:
* `Model.rigid_contact_torsional_friction`
* `Model.rigid_contact_rolling_friction`

https://github.com/newton-physics/newton/blob/main/newton/sim/model.py#L265-L268
This is exactly how XPBD is also using these numbers to scale `mu`.
You can set them to 0.05 by default if you think this is more physically realistic.